test/container-script.sh

```diff
@@ -3,5 +3,12 @@
 set -e
 
 cd /io
-pip3 install $PIP_EXTRA_ARGS $PACKAGE_LIST
+python3 -m venv .test
+source .test/bin/activate
+pip3 install --progress-bar off --upgrade pip
```

Comment:
This will result in a pip upgrade for every wheel test. That might make it run longer.